library/styleguideComponents/Wrapper/index.js

```diff
@@ -70,10 +70,63 @@ const cyanDarkTheme = {
     },
 };
 
+const pageStatus = {
+    pageName: window.location.hash.split('/')[1],
+    examplesCount: 0,
+    examplesThemes: [],
+};
+
+function registerExample() {
+    const index = pageStatus.examplesCount * 2 + 1;
+    if (!pageStatus.examplesThemes[index]) {
+        pageStatus.examplesThemes[index] = undefined;
+    }
+    pageStatus.examplesCount += 1;
+    return index;
+}
+
+function reset() {
+    pageStatus.examplesThemes = [];
+    pageStatus.examplesCount = 0;
+}
+
+function getThemeFromCache(id) {
+    const index = (Number(id) - 1) / 2;
+    return pageStatus.examplesThemes[index] || undefined;
```

Comment:
Not need of make || undefined since pageStatus.examplesThemes[index] will return undefined if it does not exists